src/radical/pilot/pmgr/launching/base.py

```diff
@@ -421,6 +421,13 @@ def _start_pilot_bulk(self, resource, schema, pilots):
 
             for fname in ru.as_list(pilot['description'].get('input_staging')):
                 base = os.path.basename(fname)
+                # checking if input staging file exists
+
+                assert os.path.exists(fname) == False
```

Comment:
If am am sure if I should use assert here, or a check like just below, raising a RuntimeError 
Please let me know, so I'll remove one of them.

Reply:
An exception will leave more detailed traces in the logs and will make it easier to understand what went wrong.